inspirehep/modules/records/wrappers.py

```diff
@@ -43,6 +43,28 @@ def admin_tools(self):
 class LiteratureRecord(ESRecord, AdminToolsMixin):
     """Record class specialized for literature records."""
 
+    def set_links_for_ids(self, extid, ext_sys_id_info):
+        """Urls and names for external system identifiers
+
+        Returns a dictionary that contains the name of the
+        external_system_identifier as the key and it's url as the value
+        """
+        if ext_sys_id_info[0] == 'KEK scanned document':
+            extid = extid.replace("-", "")
+            if len(extid) == 7 and not extid.startswith('19') and not extid.startswith('20'):
+                year = '19' + extid[:2] + '/'
+            elif len(extid) == 9:
+                year = extid[:4] + '/'
+                extid = extid[2:]
+            else:
+                return
```

Comment:
What happens when you `return None` ? Probably the code that calls this function will break

Reply:
I checked this with @jacquerie.  He suggests that since, at present there is no record with kek links other than 7 or 9 digits long, there's no need to check for them.